CONTRIBUTING.md

```diff
@@ -61,7 +61,6 @@ After you create a pull request, a bot will comment with a link to a development
 You will find a playground for the widgets: https://**netlify-deploy-url**/stories.
 
 - Example: https://deploy-preview-3376--instantsearchjs.netlify.com/stories/
```

Comment:
If there's now only one item in the list, we can inline it in the line above.

Reply:
https://github.com/algolia/instantsearch.js/pull/4280/commits/251433a6a8e6e6a71a1585a3f1c9e8fa4625e449